tests/containers/podman_netavark.pm

```diff
@@ -176,8 +175,8 @@ sub run {
     record_info('TEST3', 'create a dual stack network');
     $net1->{name} = 'test_dual_stack';
     assert_script_run("podman network create --ipv6 --gateway $net1->{gateway_v6} --subnet $net1->{subnet_v6} --gateway $net1->{gateway} --subnet $net1->{subnet} $net1->{name}");
-    assert_script_run("podman run --network $net1->{name} -d --name $ctr1->{name6} --ip6 $ctr1->{ip6} -p 8080:80 $ctr1->{image}");
-    assert_script_run("podman run --network $net1->{name} -d --name $ctr1->{name} --ip $ctr1->{ip} -p 8888:80 $ctr1->{image}");
+    assert_script_run("podman run --network $net1->{name} -d --name $ctr1->{name6} --ip6 $ctr1->{ip6} -p 8080:80 -v \$PWD:/etc/nginx/nginx.conf $ctr1->{image}");
```

Comment:
```suggestion
    assert_script_run("podman run --network $net1->{name} -d --name $ctr1->{name6} --ip6 $ctr1->{ip6} -p 8080:80 -p [::]:8080:80 -v \$PWD:/etc/nginx/nginx.conf $ctr1->{image}");
```

If we're doing dual stack, let's also listen to IPv6.

Reply:
> If we're doing dual stack, let's also listen to IPv6.

I wonder how it was working before... =/